Source/JavaScriptCore/yarr/YarrJITRegisters.h

```diff
@@ -71,6 +71,9 @@ struct YarrJITDefaultRegisters {
     static constexpr GPRReg unicodeAndSubpatternIdTemp = ARM64Registers::x5;
     static constexpr GPRReg initialStart = ARM64Registers::x11;
 
+#define UNICODE_CAN_INCREMENT_INDEX_FOR_NON_BMP 1
```

Comment:
Instead, let's define it in `PlatformEnable.h` as `ENABLE(YARR_JIT_UNICODE_CAN_INCREMENT_INDEX_FOR_NON_BMP)`

Reply:
Fixed.